### ISSUE TITLE
marker generator issue

### ISSUE THREAD

**Issue by Bruce0418** [Angelo8828/sample-arjs-implementation#1]
can you provide me with your maker generator tool that you have customized for reducing black borders to 0.9 from 0.5? you changed the codes in the armarkercontrols.js and marker trainer, so new generator should be needed. thanks in advance.

**Comment by Angelo8828** [comment]
Yes, I am very sorry about that. Right now, the borders are at 0.7 because my product manager requested me to do so. I can create a new branch by Monday if you want. :)

Sorry because I'm out of town right now and I cannot do right now your request.

**Comment by Angelo8828** [comment]
Please see this commit on my AR.js project copy. Please reduce the *blackMargin* variable to **0.04** in the threex-arpatternfile.js or maybe you can just clone my work and reset to that specific commit.

https://github.com/Angelo8828/AR.js/commit/11fd8f4514912ec5d0d28791d3a340b86dd94aad

**Comment by Bruce0418** [comment]
Angelo,

Thanks for your reply.
I really appreciate you to share this sample-arjs-implementation.
I was using your js files in this repository and was able to test with my maker image, not a hiro marker but my marker generator page doesn't work correctly eventually. 
https://bruce0418.github.io/AR.js/three.js/examples/marker-training/examples/generator.html
yes it's missing uploaded image and goes fail to download patt file.
Do you have any idea how it could be fixed?

I followed your instructions left here to build my own repository for reducing black border:
https://github.com/jeromeetienne/AR.js/issues/296
so I revised the threex-arprtternfile.js and three-armarkercontrols.js as
you suggested.
and this is my repo: https://github.com/Bruce0418/AR.js

Hope you help me out soon:)
Thanks much

Bruce
